src/Agent/NewRelic/Profiler/ConfigurationTest/InstrumentationConfigurationTest.cpp

```diff
@@ -73,6 +73,205 @@ namespace NewRelic { namespace Profiler { namespace Configuration { namespace Te
             Assert::IsFalse(instrumentationPoint == nullptr);
         }
 
+        TEST_METHOD(basic_match_with_version)
+        {
+            InstrumentationXmlSetPtr xmlSet(new InstrumentationXmlSet());
```

Comment:
Should we use `make_shared` here instead of the `new` keyword?

Reply:
Probably. I was sticking with the convention the other tests in the class were using.